Sources/CovidCertificateSDK/TrustList/TrustCertificatesUpdate.swift

```diff
@@ -0,0 +1,70 @@
+//
+/*
+ * Copyright (c) 2021 Ubique Innovation AG <https://www.ubique.ch>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Foundation
+
+class TrustCertificatesUpdate: TrustListUpdate {
+    // MARK: - Session
+
+    let session = URLSession.shared
+
+    // MARK: - Update
+
+    override internal func synchronousUpdate() -> NetworkError? {
+        // update active certificates service
+        let requestActive = CovidCertificateSDK.currentEnvironment.activeCertificatesService.request()
+        let (dataActive, _, errorActive) = session.synchronousDataTask(with: requestActive)
+
+        if errorActive != nil {
+            return errorActive?.asNetworkError()
+        }
+
+        guard let d = dataActive, let result = try? JSONDecoder().decode(ActiveTrustCertificates.self, from: d) else {
+            return .NETWORK_PARSE_ERROR
+        }
+
+        _ = trustStorage.updateActiveCertificates(result)
+
+        // update trust certificates service
+        var listNeedsUpdate = true
+
+        while listNeedsUpdate {
+            let request = CovidCertificateSDK.currentEnvironment.trustCertificatesService(since: trustStorage.certificateSince()).request()
+            let (data, response, error) = session.synchronousDataTask(with: request)
+
+            if error != nil {
+                return error?.asNetworkError()
+            }
+
+            // get the x-next-since, save it as well and pass to the next request
+            var nextSinceHeader: Int64 = 0
+            if let s = (response as? HTTPURLResponse)?.allHeaderFields["x-next-since"] as? String,
+               let nextHeader = Int64(s) {
```

Comment:
I think we should not convert `x-next-since` to a `Int` but leave it as `String`. For the client it doesn't really matter (does not need to be interpreted) and it allows the backend to pass any other format in there (if ever needed).